docs/CYPRESS.md

```diff
@@ -30,20 +35,59 @@ Plio uses `cypress.env.json` to set up environment variables for the Cypress app
 
 #### Cypress ENV variables
  Below are the details of all the environment variables.
-##### `google_auth.refresh_token`
+##### `auth_google_refresh_token`
 Google refresh token retrieved in the pre-requisites.
 
-##### `google_auth.client_id`
+##### `auth_google_client_id`
 Google client ID retrieved in the pre-requisites.
 
-##### `google_auth.client_secret`
+##### `auth_google_client_secret`
 Google client secret retrieved in the pre-requisites.
 
-##### `plio_backend.convert_social_auth_token_url`
+##### `plio_backend_convert_social_auth_token_url`
 The convert token URL in the Plio backend repo that converts the Social Auth Token and returns a valid Django Auth Token. By default, the value is `http://0.0.0.0:8001/auth/convert-token`.
 
-##### `plio_backend.client_id`
+##### `plio_backend_client_id`
 The client id for backend API application.
 
-##### `plio_backend.client_secret`
+##### `plio_backend_client_secret`
 The client secret for backend API application.
+
+
+### Cypress Dashboard
+The Cypress Dashboard is a web interface that allows you to run and debug your Cypress tests. To integrate Cypress Dashboard with your local integration tests, follow the steps below:
+1.  Create an account in [Cypress Dashboard](https://dashboard.cypress.io/)
+2.  Create a project if not already exists
+3.  Get the projectId from project settings page and replace the `projectId` in `cypress.json` with the projectId.
+4.  Get the Cypress Record Key from project settings page.
+5.  Run the following command. This will record the integration tests and upload the results to the Cypress Dashboard.
+    ```sh
+    npx cypress run --record --key your_cypress_key
+    ```
+
+### Continuous Integration
+The file [integration-testing.yml](../.github/workflows/integration-testing.yml) is used to run the integration tests in GitHub Actions.
+To set up CI, follow the steps below:
+1. Create a new GitHub Environment and name it `Integration Testing`.
+2. Add the following variables to the GitHub environment:
+   1. Cypress Dashboard - Refer Cypress Dashboard section above
+      1. `CYPRESS_RECORD_KEY`
+   2. Plio Analytics (CubeJS) - Refer [Plio Analytics ENV guide](https://github.com/avantifellows/plio-analytics/blob/master/docs/ENV.md)
+      1. `CUBEJS_API_SECRET`
+   3. Plio Frontend (Cypress) - Refer Frontend Cypress ENV guide above
+      1. `CYPRESS_AUTH_GOOGLE_CLIENT_ID`
+      2. `CYPRESS_AUTH_GOOGLE_CLIENT_SECRET`
+      3. `CYPRESS_AUTH_GOOGLE_REFRESH_TOKEN`
+      4. `CYPRESS_PLIO_BACKEND_CONVERT_SOCIAL_AUTH_TOKEN_URL`
+   4. Plio Frontend (VueJS) - Refer [Plio Frontend ENV guide](../docs/ENV.md)
+      1. `VUE_APP_BACKEND_API_CLIENT_ID`
+      2.  `VUE_APP_BACKEND_API_CLIENT_SECRET`
+      3.  `VUE_APP_GOOGLE_CLIENT_ID`
+      4.  `VUE_APP_I18N_LOCALE`
```

Comment:
shouldn't we list all the environment variables that we use for frontend? otherwise, every time that we write a test case that uses a new environment variable, we'll have to update this doc too

Reply:
I don't think we're leaving any environment variable that's required in the running of the application. Many of them have default values in `.env.example` so we're skipping them here.

Do you see anything that we need to add here? I was thinking with less env variables, it'll be faster (and more comfortable) to set up.